document-generator-accounts/src/main/java/uk/gov/companieshouse/document/generator/accounts/data/accounts/AccountsManager.java

```diff
@@ -17,9 +24,14 @@
 @Component
 public class AccountsManager {
 
+    private static final String SMALL_FULL = "/small-full";
+
     @Autowired
     private ApiClientService apiClientService;
 
+    @Autowired
+    private CompanyService companyService;
```

Comment:
I would use constructor injection in here.

Reply:
Is there a specific advantage to doing it this way?

Reply:
No advantage, it's Spring's recommendation